freegames/madlibs.py

```diff
@@ -0,0 +1,59 @@
+""" Mad Libs: Funny Story Creation Game.
+
+Exercises:
+
+1. Can you create functionality to replace the given story?
+2. Can you load the story and template from a file?
+3. Can you add additional tenses?
+4. Can you add flexibility in a loaded story template?
+5. Can you adjust for grammar? (past tense, etc.)
+
+"""
+
+story = "The [adj] [adj] [n] [v] over the [adj] [adj] [n]"
+# The quick black fox jumped over the lazy brown dog.
```

Comment:
The [pangram](https://en.wikipedia.org/wiki/The_quick_brown_fox_jumps_over_the_lazy_dog) is actually: "The quick brown fox jumps over the lazy dog"

```suggestion
# The quick brown fox jumps over the lazy dog.
```